gcloud-java-storage/src/main/java/com/google/cloud/storage/testing/FakeStorageRpc.java

```diff
@@ -37,9 +37,24 @@
 
 /**
  * A bare-bones in-memory implementation of Storage, meant for testing.
- * See LocalGcsHelper.
  *
- * This class is NOT thread-safe.
+ * <p>This class is <i>not</i> thread-safe. It's also (currently) limited in the following ways:
+ * <ul>
+ * <li>Partially Supported
+ *   <ul>
+ *   <li>create
+ *   <li>get
+ *   <li>delete
+ *   <li>directory listing
+ *   </ul>
+ * <li>Unsupported
```

Comment:
As to the first comment, that's exactly what I meant by partially supported. I [pushed](https://github.com/GoogleCloudPlatform/gcloud-java/pull/1070) a change to split those apart in the description.